openupgradelib/openupgrade.py

```diff
@@ -2372,6 +2372,15 @@ def format_message(f):
         globaldict = {'uid': env.uid}
         if version_info[0] < 14:
             globaldict.update({'time': time})
+        if version_info[0] >= 13:
```

Comment:
```suggestion
        if version_info[0] > 12:
```

Reply:
Current one is the same valid, as we are getting the main version only (`version_info[0]`)

Reply:
On that mood, I prefer to see `13`, that is the significant version that contains already the change.